setup.cfg

```diff
@@ -0,0 +1,45 @@
+[metadata]
+name = pysize
```

Comment:
Project/package distribution with this name already exists on PyPI: https://pypi.org/p/pysize. The upload will fail should you try to reuse it.
Let's use smth else like `pysize-inspector`.

cc @bosswissam ^